src/com/lftechnology/java/training/srijan/basics/stemandleaf/DataPresentation.java

```diff
@@ -0,0 +1,190 @@
+package com.lftechnology.java.training.srijan.basics.stemandleaf;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.logging.Logger;
+
+public class DataPresentation {
+	private int[] datas;
+	private static final Logger LOGGER = Logger.getLogger(DataPresentation.class.getName());
+	private int[][] stemFrequencyLeaf;
+	List<Integer> stemAndLeaf = new ArrayList<>();
+	Set<Integer> stem = new TreeSet<>();
+	List<Integer> leaf = new ArrayList<>();
+
+	/**
+	 * <p>
+	 * constructor
+	 * </p>
+	 * <p>
+	 * initialize array
+	 * </p>
+	 * 
+	 * @param numberOfDatas
+	 *            {@link Integer}
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public DataPresentation(int numberOfDatas) {
+		datas = new int[numberOfDatas];
+		stemFrequencyLeaf = new int[numberOfDatas][3];
+		for (int i = 0; i < datas.length; i++) {
+			datas[i] = 0;
+		}
+
+		for (int i = 0; i < numberOfDatas; i++) {
+			for (int j = 0; j < 3; j++) {
+				stemFrequencyLeaf[i][j] = 0;
+			}
+		}
+	}
+
+	/**
+	 * <p>
+	 * controls all the Methods
+	 * </p>
+	 * <p>
+	 * Inputs Data, display unsorted Data, sorts input Data
+	 * </p>
+	 * 
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public void dataHandler() {
+		inputDataPoints();
+		displayUnsortedInputData();
+		insertionSort();
+		separateStemFrequencyLeaf();
+	}
+
+	/**
+	 * <p>
+	 * Inputs Data and stores in an array
+	 * </p>
+	 * 
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public void inputDataPoints() {
+		Scanner inputData = new Scanner(System.in);
+		LOGGER.info("Input Datas");
+		try {
+			for (int i = 0; i < datas.length; i++) {
+				datas[i] = inputData.nextInt();
+				stemAndLeaf.add(datas[i]);
+				// treeSetOfStemAndLeaf.add(datas[i]);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)